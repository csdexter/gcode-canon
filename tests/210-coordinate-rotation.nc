(testing coordinate systems)
(machine setup)
M05 S600
M09
M23
M49
M69

(turn servos ON)
M17

(control setup)
G15
G17
G23
G40
G49
G50
G69

(in mm)
G21

(absolute coordinates)
G90

(rotation)
G01 X10 Y10 Z10 F600
X15
Y12 X8
X10 Y10
(rotate 90 degrees and do it again)
G68 R90
X15
Y12 X8
X10 Y10
(disable rotation)
G69

(and now for something more exotic)


M02