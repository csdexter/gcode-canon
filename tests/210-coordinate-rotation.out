MSG: WAR: Machine servos activated!
MSG: STA: Scanning input for programs (O words)
MSG: WAR: Machine servos activated!
MPOS,10.00,10.00,10.00
MPOS,15.00,10.00,10.00
MPOS,8.00,12.00,10.00
MPOS,10.00,10.00,10.00
MPOS,10.00,15.00,10.00
MPOS,8.00,8.00,10.00
MPOS,10.00,10.00,10.00
MPOS,20.00,10.00,10.00
MPOS,25.00,10.00,10.00
MPOS,25.00,12.50,10.00
MPOS,15.00,12.50,10.00
MPOS,15.00,7.50,10.00
MPOS,25.00,7.50,10.00
MPOS,25.00,10.00,10.00
MPOS,20.00,10.00,10.00
MPOS,20.61,20.61,10.00
MPOS,18.84,22.37,10.00
MPOS,11.77,15.30,10.00
MPOS,15.30,11.77,10.00
MPOS,22.37,18.84,10.00
MPOS,20.61,20.61,10.00
MPOS,17.07,17.07,10.00
MPOS,2.50,22.99,10.00
MPOS,0.33,21.74,10.00
MPOS,5.33,13.08,10.00
MPOS,9.67,15.58,10.00
MPOS,4.67,24.24,10.00
MPOS,2.50,22.99,10.00
MPOS,5.00,18.66,10.00
